tutorials/asr/Confidence_Ensembles.ipynb

```diff
@@ -0,0 +1,517 @@
+    "2. The runtime cost grows linearly with each added model, which limits the practically useful ensemble size.\n",
+    "3. Given enough compute and data, it is likely possible to build specialized models that would outperform confidence-based ensembles on most tasks.\n",
+    "\n",
+    "To sum up — if you're combining a small number of models (e.g., up to 5), can afford a few seconds of additional latency and don't have resources to build a specialized model, confidence-based ensembles might be a good fit and you should try them out! There are many ASR models that you can combine in the ensemble available in [NVIDIA NGC cloud](https://catalog.ngc.nvidia.com/models) as well as other model hubs, such as [Hugging Face](https://huggingface.co/nvidia).\n",
+    "\n",
+    "In the next few cells we will cover what a confidence-based ensemble is and some best practices of using these models. Each cell is mostly self-contained, so feel free to skip around or jump directly to the code part if you want to see usage examples right away."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## What is a confidence-based ensemble?\n",
+    "\n",
+    "You're probably familiar with more traditional [ensembles of machine learning models](https://en.wikipedia.org/wiki/Ensemble_learning). Confidence ensembles are a less popular approach where we only use an output of a single model that is deemed best for the current input. A typical way to pick the \"best\" output is to select a model with the highest confidence score, which provides an estimate of how likely the output is to be correct. Here is a schematic illustration of the model.\n",
+    "\n",
+    "<img src=\"https://github.com/NVIDIA/NeMo/releases/download/v1.19.0/conf-ensembles-overview.png\" alt=\"Confidence-ensemble schematic representation\" width=\"600\"/>\n",
+    "\n",
+    "As you can see, to define confidence ensemble, we need to define 3 things:\n",
+    "\n",
+    "1. Which models are part of the ensemble.\n",
+    "2. How do we estimate model's confidence.\n",
+    "3. How do we \"calibrate\" confidence values via a model selection block.\n",
+    "\n",
+    "Let's discuss each of these 3 items below.\n",
+    "\n",
+    "### Which models to use?\n",
+    "\n",
+    "A short answer — you can use any ASR models. E.g., you can combine a number of CTC models, or Transducer models, or even mix-and-match. \n",
+    "\n",
+    "A more detailed answer is that hte performance of the confidence ensemble is upper-bounded by the performance of the best model on each of the input examples. Thus you will benefit if some of your models work really well on part of the input compared to other models. This way you will get more gains compared to each separate model, and it will also make correct model identification easier.\n",
```

Comment:
Imo this section is quite heavy to read in one sitting. Breaking apart the subsection into digestible chunks may be better c

This is fine as well, just a bit heavy.

Reply:
I was not sure how much details to put, so decided to put enough for people to understand a bit more than just examples, but also encourage to skip around the text and go straight to code and return if they want to learn more. Maybe I should make it more explicit - right now I just say 

> Each cell is mostly self-contained, so feel free to skip around or jump directly to the code part if you want to see usage examples right away.

at the end of the first text cell.